app/views/listings/show.html.haml

```diff
@@ -0,0 +1,15 @@
+%h1= @listing.pet_name
+%p=@listing.pet_picture
+%p Dates: #{@listing.start_date} to #{@listing.end_date}
+
+%h2 Offers
+- if @listing.offers.any?
+  - @listing.offers.each do |offer|
+    .offer{id: dom_id(offer)}
+    %p= offer.name
+    %p= offer.email
+    %p= offer.location
+    %p #{offer.price} kr
+    %button= link_to "Accept offer", "#"
```

Comment:
You can't have a link in a button. Hängslen & livrem? 🤪

Reply:
Ah yes :) Changed this to only be a link, to also be more consistent with index page and other links.